.github/workflows/docker.yaml

```
@@ -0,0 +1,42 @@
+# https://docs.github.com/en/actions/deployment/deploying-to-your-cloud-provider/deploying-to-amazon-elastic-container-service
+name: Build and push docker image
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy:
+    name: Deploy
```

Since we're not deploying the application with this GHA workflow.
```suggestion
  docker:
    name: Build docker image
```